catalog/resource_volume.go

```diff
@@ -82,7 +87,11 @@ func ResourceVolume() common.Resource {
 			if err != nil {
 				return err
 			}
-			return common.StructToData(v, s, d)
+			err = common.StructToData(v, s, d)
+			if err != nil {
+				return err
+			}
+			return d.Set("volume_path", "/Volumes/"+strings.ReplaceAll(v.FullName, ".", "/"))
```

Comment:
we also need to update this in Update, as volume name could change

Reply:
done, thank you!